docs/docker-integration/cloudformation.rst

```diff
@@ -0,0 +1,167 @@
+Installing Flocker with Swarm on AWS using CloudFormation
+=========================================================
+
+The steps in this guide enable you to quickly deploy a Flocker/Swarm cluster, as illustrated below, which will create four EC2 instances on your AWS account:
+
+.. raw:: html
+
+	<div style="width:80%; margin-left:auto; margin-right:auto; margin-bottom:2em;">
+
+.. figure:: ../images/cloudformation.png
+    :alt: A diagram illustrating a cluster of four AWS EC2 instances running Flocker with Docker Swarm.
+
+.. raw:: html
+
+	</div>
+
+Follow the steps below to create your cluster.
+Once it's up and running we'll guide you through a tutorial to deploy a sample app.
+
+.. source material for this image: https://drive.google.com/open?id=0ByymF9bLBknGeXlPX1pTdXVZOGM
+
+.. raw:: html
+
+	<div class="step-stages step-stages--3up">
+		<div class="step-stages__excerpt">
+			<h2 class="step-stages__heading">Step 1</h2>
+			<p>Create an AWS Key Pair for secure login to your cluster.</p>
+		</div>
+		<div class="step-stages__step first">
+			<img src="../_images/01-keys-menu.png" alt="AWS key pairs section in console"/>
+            <span><a href="https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#KeyPairs:sort=keyName" target="_blank">Log in to the AWS console</a>. This will open "N. Virginia" region, "Key Pairs" section.</span>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/02-create-key.png" alt="Creating a new AWS key pair"/>
+            <span>Click "Create Key Pair". Give your key pair a meaningful name, like <strong>flocker-test</strong>. You'll need this later.</span>
+		</div>
+		<div class="step-stages__step">
+			<img src="../_images/03-pem-downloaded.png" alt="A downloaded pem file"/>
+            <span>The private key (.pem file) will be downloaded onto your computer.</span>
+		</div>
+	</div>
+
+	<div class="step-stages step-stages--3up">
+		<div class="step-stages__excerpt">
+			<h2 class="step-stages__heading">Step 2</h2>
+			<p>Create a Flocker cluster using our CloudFormation template:
+            <br />
+			<a href="https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?templateURL=https:%2F%2Fs3.amazonaws.com%2Finstaller.downloads.clusterhq.com%2Fflocker-cluster.cloudformation.json" class="button" target="_blank" align="middle">Launch Flocker CloudFormation</a>
+            <br />
+			This button will open CloudFormation in a new window.</p>
+		</div>
+		<div class="step-stages__step first">
+			<img src="../_images/11-cloudformation-stackname.png" alt="Specifying the stack name"/>
+			<span>Click "Next". Enter a <code>Stack name</code>. This can be any descriptive name.</span>
+			<span>Enter your <code>KeyName</code> from Step 1. Then enter your AWS <code>AccessKeyID</code> and <code>SecretAccessKey</code> credentials.</span><span>If you don't know these, <a href="javascript:void(0);" onclick="$('#iam-instructions').show();">click here</a>.</span>
```

Comment:
> If you don't know these, <a href="javascript:void(0);" onclick="$('#iam-instructions').show();">click here</a>.</span>

Can we please consider saying "If you do not know these, learn more about managing AWS API keys (with an href to
 https://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html)".

Thanks.